tests/e2e/kubetest2-kops/aws/zones.go

```diff
@@ -75,7 +75,6 @@ var allZones = []string{
 	"us-east-2a",
 	"us-east-2b",
 	"us-east-2c",
-	"us-west-1a",
 	"us-west-1b",
```

Comment:
Please just comment these same as the one below. We can live without any `us-west-1`.

Reply:
Is there a benefit to comment them out?

Reply:
Yes. At some point someone will wonder why there's no `us-west-1` and he will add it, then remove it again.
This way maybe he reads the comment first :D.

Reply:
For me 1c exists. It probably depends on the account which one gets cut.
I'll leave this change out for now.